gems/smithy/lib/smithy/templates/client/schema.erb

```diff
@@ -15,11 +15,14 @@ module <%= module_name %>
 <% shape.members.each do |member| -%>
     <%= shape.name %>.<%= member.add_member_method(shape.type) %>
 <% end -%>
+<% if shape.type == 'UnionShape' -%>
+    <%= shape.name %>.add_member(:unknown, 'unknown', Prelude::Unit, Types::<%= shape.name %>::Unknown)
```

Comment:
I forgot to ask - I wasn't sure what the "shape" representation of Unknown member should be since I need to put something in there but went with `Prelude::Unit` since it is used to represented as `nil` in smithy models but open to alternatives

Reply:
Good question. I think it may actually be a union shape?